core-web/libs/sdk/experiments/tsconfig.json

```diff
@@ -1,16 +1,24 @@
 {
     "extends": "../../../tsconfig.base.json",
     "compilerOptions": {
-        "module": "commonjs",
-        "forceConsistentCasingInFileNames": true,
+        "target": "ESNext",
+        "useDefineForClassFields": true,
+        "module": "ESNext",
         "strict": true,
-        "noImplicitOverride": true,
-        "noPropertyAccessFromIndexSignature": true,
-        "noImplicitReturns": true,
-        "noFallthroughCasesInSwitch": true
+        "moduleResolution": "Node",
+        "resolveJsonModule": true,
+        "isolatedModules": true,
+        "types": ["vite/client"],
+        "noEmit": true,
+        "lib": ["ESNext", "DOM"],
+        "skipLibCheck": true,
```

Comment:
Why this one?

Reply:
nothing important, I think. "Skips type checking for type declaration files of all imported libraries (*.d.ts). This can speed up compilation in large projects."

Reply:
None of the other libraries have all this much properties in their respective `tsconfig`, let's not put things we don't understand, less is better.